lib/sign_in/widgets/form.dart

```diff
@@ -2,41 +2,32 @@ import 'package:flutter/material.dart';
 import 'package:flutter_svg/svg.dart';
 import 'package:pedantic/pedantic.dart';
 import 'package:schulcloud/app/app.dart';
+import 'package:webview_flutter/webview_flutter.dart';
 
 import '../bloc.dart';
-import 'input.dart';
-import 'morphing_loading_button.dart';
 
 class SignInForm extends StatefulWidget {
   @override
   _SignInFormState createState() => _SignInFormState();
 }
 
 class _SignInFormState extends State<SignInForm> {
-  final _emailController = TextEditingController();
-  final _passwordController = TextEditingController();
-  bool _isEmailValid = true;
-  bool _isPasswordValid = true;
+  WebViewController controller;
 
   bool _isLoading = false;
   String _ambientError;
 
-  Future<void> _executeSignIn(Future<void> Function() signIn) async {
+  Future<void> _executeLogin(Future<void> Function() login) async {
     setState(() => _isLoading = true);
 
     try {
-      await signIn();
+      await login();
```

Comment:
These and related functions were renamed to use "signIn" instead of "login" in their names recently, but probably the merge went wrong. Can you please adopt the new naming?